### ISSUE TITLE
estimateGas tests fail

### ISSUE THREAD

**Issue by aakilfernandes** [ConsenSys-archive/ganache-cli-archive#222]
Output:
https://gist.github.com/aakilfernandes/5e7c866d38bb3b5842c92725f0995184

## Your Environment
Mac
node v6.3.0
testrpc@3.0.3#00d3eba50086a4163e39e1292bc2e0e6ce4269c9

**Comment by tcoulter** [comment]
We recently upgraded the compiler version and tacked the release onto a specific version. Can you blow away your `node_modules` directly and reinstall? Because it's tacked to a specific version the gas costs should remain the same.

**Comment by aakilfernandes** [comment]
Reinstalled and now got a bunch more errors =/. looks like someone forgot to bump =)

https://gist.github.com/aakilfernandes/28c46db9b07b27a3a4a52082a558b635

**Comment by tcoulter** [comment]
Ya, a dependency updated, still trying to figure out which one.

**Comment by tcoulter** [comment]
Those test failures should be fixed here (travis currently running). https://github.com/ethereumjs/testrpc/pull/225

Once #225 is merged, blow away your `node_modules` directory again and reinstall, as fixes have been added upstream.

**Comment by tcoulter** [comment]
Alright, merged into `master`. Repull from master and then reinstall dependencies.

**Comment by aakilfernandes** [comment]
Confirmed fixed. Thanks @tcoulter 💪 !
